CONTRIBUTING.md

```diff
@@ -14,6 +14,21 @@ To use `docker-compose` from source code, use the
 $ ln -sf docker-compose.yml.dev docker-compose.yml
 ```
 
+## To run the tests
+
+To run the Python test, make sure an elasticsearch service is running on localhost,
+for example by executing:
+
```

Comment:
I think we should tell, that:

Tests rely on the Elasticsearch service so first you need to ensure the Elasticsearch container is running. You can run the "docker-compose up -d" or you can only run the Elasticsearch container by running "docker-compose start elastic".

Reply:
Ok, do you mind keeping the podman command for user without docker?

Reply:
+1